### PR TITLE
Editing

### PR DESCRIPTION
**Pull request by JohannesHentrich**
Editing manual/python/index.md
- Removing passive voice
- Removing numbering from the "Loading user libraries and Just-In-Time compilation (JITting)" subsections. The single subchapters are not consecutive and can be read independently.
- Improving readability (hopefully doing so).

### CODE REVIEW

**Comment by couet** [comment]
Hi @JohannesHentrich it looks fine for me. As there is quite a lot of changes, I guess we should wait @etejedor approval
 before merging.

**Review by couet — APPROVED** [review]

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:10]: Here I would like to make it clear that PyROOT is the Python bindings, so perhaps something like `With PyROOT, ROOT's Python-C++ bindings, you can use ROOT from Python`?

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:16]: I wouldn't say `in-depth knowledge` is required, at least for the most common operations. I think `working knowledge` is fine or alternatively `fair knowledge`.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:105]: I like it better how it was before, `If PyROOT is terminated during its cleanup phase` does not say what I meant. What about this: `When the application finishes, during PyROOT's cleanup phase, the ROOT C++ interpreter is shut down by default.`.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:156]: In this paragraph, what about this modification: "For example, by calling `SetOwnership(obj, False)`, you make sure ...". It is by using `SetOwnership` that you make sure the C++ object is not deleted.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:164]: I think that the fact that the options are ordered by complexity and performance justifies the numbering below. (instead of just bullets).

File: manual/python/index.md
Comment (etejedor) [reply manual/python/index.md:390]: In Jupyter notebooks, cells have an "output" section below them, where the output is displayed. I think it's fine.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:424]: I think the `, which` is what I meant here: I am providing some extra information about class templates, I am not specifying which class templates are not affected.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:573]: With the `:` here I meant to explain how the functionality is automatically provided by new cppyy.

File: manual/python/index.md
Comment (etejedor) [inline manual/python/index.md:633]: Same comment here about the `:`, perhaps it's a language difference? I use `:` to link with a clarification/further explanation of my previous statement.

**Review by etejedor — CHANGES_REQUESTED** [review]
Thanks a lot @JohannesHentrich ! Please see my comments.

File: manual/python/index.md
Comment (hageboeck) [inline manual/python/index.md:163]: `high-performance C++` was the correct spelling.

**Comment by hageboeck** [comment]
> Removing numbering from the "Loading user libraries and Just-In-Time compilation (JITting)" subsections. The single subchapters are not consecutive and can be read independently.

I actually numbered them so people can select one approach from the list of alternatives above, and then jump down and *only* read "their" number. I'm not strongly attached to those numbers, though. Just pointing out that I instinctively did the exact opposite, but with the same intention.

File: manual/python/index.md
Comment (JohannesHentrich) [reply manual/python/index.md:105]: Your proposal is indeed clearer. I think that the term "is terminated" is somehow more technical than "finishes", but may this is only a matter of taste.

File: manual/python/index.md
Comment (JohannesHentrich) [reply manual/python/index.md:573]: sure, but then it a very long sentence. But anyhow "when" must be reflaced by "if".

File: manual/python/index.md
Comment (JohannesHentrich) [reply manual/python/index.md:424]: I know that people like using "which". But in written English "that" is used and I think every proofreader would change "which" to "that" ( and I can't count the times when it was changed on me :-)) But we can let it as is was. But we can let it as is was

File: manual/python/index.md
Comment (JohannesHentrich) [reply manual/python/index.md:164]: An ordered list is only used, if there is a sequence (especially in tasks). Complexity (which means for everybody something else) is no real ordering criteria (in the Technical Documentation).

if you only scan the menu on the left side, it is assumed that there is a sequence to be followed:
   Loading user libraries and Just-In-Time compilation (JITting)
     1. Just-in-time compilation ...
     2. Just-in-time compilation ..
     3. Loading ...
     4. Loading ...

So in this case a numbered list is not correct.

File: manual/python/index.md
Comment (etejedor) [reply manual/python/index.md:424]: Thanks for the explanation, but if I can't use `which`, how can I express addition of information? My point is what is explained here: https://grammar.yourdictionary.com/grammar-rules-and-tips/when-to-use-which-or-that.html . I don't want to be restrictive in what class templates I'm talking about, I want to add extra info about class templates.

**Comment by couet** [comment]
It seems we can merge now, can't we ?

**Comment by eguiraud** [comment]
Standard practice would be to wait for a green light by the reviewers that requested changes

**Review by etejedor — APPROVED** [review]
LGTM!

**Pull request merged**
